docs/applications/big-data/how-to-move-machine-learning-model-to-production.md

```diff
@@ -0,0 +1,306 @@
+
+{:.note}
+> This model is simple enough, and the data set small enough, that the script can be run on a Linode or on your local machine. However, using a computer without a GPU will still take at least ten minutes. If you would prefer to skip this step, a pre-trained model can be downloaded by running the command `wget https://github.com/linode/docs-scripts/raw/master/hosted_scripts/my_model.h5` 
+>
+> Older versions of Keras requires deleting optimizer weights in the pre-trained model. If the pre-trained model is downloaded from Github, the script below checks and removes optimizer weights.
+>
+>     import h5py
+>     with h5py.File('my_model.h5', 'r+') as f:
+>         if 'optimizer_weights' in f.keys():
+>             del f['optimizer_weights']
+>         f.close()
+>
+
+1.  Create a directory for the model:
+
+        mkdir ~/models && cd ~/models
+
+2.  Create a Python script to build and train your model:
+
+    {:.file}
+    ~/models/mnist_model.py
+    :   ~~~
+        from keras.models import Sequential
+        from keras.layers import Dense, Dropout, Activation, Flatten
+        from keras.layers import Convolution2D, MaxPooling2D
+        from keras.utils import np_utils
+        from keras.datasets import mnist
+
+
+        (X_train, y_train), (X_test, y_test) = mnist.load_data()
+        X_train = X_train.reshape(X_train.shape[0],1,28,28)
+        X_test  = X_test.reshape(X_test.shape[0],1,28,28)
+        X_train = X_train.astype('float32')
+        X_test = X_test.astype('float32')
+        X_train /= 255
+        X_test /= 255
+
+        Y_train = np_utils.to_categorical(y_train, 10)
+        Y_test = np_utils.to_categorical(y_test, 10)
+
+        model = Sequential()
+
+        model.add(Convolution2D(32,(3,3),activation='relu',input_shape=(1,28,28),dim_ordering='th'))
+        model.add(MaxPooling2D(pool_size=(2,2)))
+        model.add(Dropout(0.25))
+        model.add(Flatten())
+        model.add(Dense(128, activation='relu'))
+        model.add(Dropout(0.5))
+        model.add(Dense(10, activation='softmax'))
+        model.compile(loss='categorical_crossentropy',
+                      optimizer='adam',
+                      metrics=['accuracy'])
+
+        model.fit(X_train, Y_train,
+                  batch_size=32, nb_epoch=5, verbose=1)
+
+        model.save('my_model.h5')
+
+        ~~~
+
+3.  Run the script:
+
+        python ./mnist_model.py
+
+    There may be a warning message as shown below from a pip or conda installation which means installing from source could offer superior performance.
```

Comment:
@Jskobos this says there might be a message about installing from source over the method here, but it doesn't say to or not to. Should we suggest that the reader ignore the message?

Reply:
@EdwardAngert Sorry, just saw this. That comment was from Sam, yes the warning can be ignored.

Reply:
Idea was to clarify that it's possible to speed things up if the reader is up to it.